docs/howtos/optax_update_guide.rst

```diff
@@ -0,0 +1,249 @@
+Optax Update Guide
+==================
+
+Code samples below are executable in
+`Colab <https://colab.research.google.com/github/google/flax/blob/main/docs/notebooks/optax_update_guide.ipynb>`_.
+
+
+.. testsetup::
+
+  import flax
+  import jax
+  import jax.numpy as jnp
+  import flax.linen as nn
+  import optax
+
+  batch = {'image': jnp.ones([1, 28, 28, 1]), 'label': jnp.array([0])}
+  ds_train = [batch]
+  get_ds_train = lambda: [batch]
```

Comment:
nit: do we need both lines? (they seem to be the same thing)

Reply:
Added a comment to clarify.